games-strategy/freeciv/freeciv-2.5.10.recipe

```diff
@@ -0,0 +1,119 @@
+SUMMARY="A free Civilization-like game"
+DESCRIPTION="Freeciv is a free Civilization-like game, primarily \
+for X under Unix. It has support for multiplayer games locally or \
+over a network, and an AI which gives most people a run for their money.
+Freeciv aims to be mostly rule-compatible with Civilization II [tm], \
+published by Sid Meier and Microprose [tm]. A few rules are different \
+where we think it makes more sense, and we have lots and lots of \
+adjustable parameters to make customizing games possible.
+Freeciv has been implemented completely independently of Civilization; \
+you do not need to own Civilization to play Freeciv."
+HOMEPAGE="http://www.freeciv.org"
+COPYRIGHT="1996-2017 The Freeciv Team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://downloads.sourceforge.net/project/freeciv/Freeciv%202.5/$portVersion/freeciv-$portVersion.tar.bz2"
+CHECKSUM_SHA256="c8a14156562dce2f0a7b96390c2d5a35314f1124c448782a6a47b0c51cfcb302"
+ADDITIONAL_FILES="freeciv.rdef.in"
+PATCHES="freeciv-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+	settings/freeciv/database.lua keep-old
+	"
+
+PROVIDES="
+	freeciv$secondaryArchSuffix = $portVersion
+	app:freeciv$secondaryArchSuffix = $portVersion
+	cmd:freeciv_manual$secondaryArchSuffix
+	cmd:freeciv_sdl$secondaryArchSuffix
+	cmd:freeciv_server$secondaryArchSuffix
```

Comment:
We should drop `$secondaryArchSuffix` and instead make use of `$commandSuffix` construct, as they're unneeded if a binary doesn't run on the target architecture. Also, `app:`s don't need `$secondaryArchSuffix`. Maybe we should consider dropping the bins on secondary architecture.

```diff
-	app:freeciv$secondaryArchSuffix = $portVersion
-	cmd:freeciv_manual$secondaryArchSuffix
-	cmd:freeciv_sdl$secondaryArchSuffix
-	cmd:freeciv_server$secondaryArchSuffix
+	app:freeciv = $portVersion
+	cmd:freeciv_manual$commandSuffix
+	cmd:freeciv_sdl$commandSuffix
+	cmd:freeciv_server$commandSuffix
```

Reply:
This doesn't seem to work ... 
  Warning: POLICY ERROR: no matching provides "cmd:freeciv_manual_x86" for "bin/freeciv-manual-x86"
  Warning: POLICY ERROR: no matching provides "cmd:freeciv_server_x86" for "bin/freeciv-server-x86"

Reply:
Unless you pass `--omit-dirs binDir` to `runConfigure` and `--bindir=$commandBinDir` (which is `$prefix/bin`) to `./configure`, it won't work.